misc/codegen/lib/schemadefs.py

```diff
@@ -278,7 +279,7 @@ def __or__(self, other: _schema.PropertyModifier):
 drop = object()
 
 
-def annotate(annotated_cls: type, replace_bases: _Dict[type, type] | None = None) -> _Callable[[type], _PropertyAnnotation]:
+def annotate(annotated_cls: type, add_bases: _List[type] | None = None, replace_bases: _Dict[type, type] | None = None) -> _Callable[[type], _PropertyAnnotation]:
```

Comment:
the type annotation can be `typing.Iterable` here.

Also, could you add a unit test in `test_schemaloader.py`? Should be easy copying and adapting `test_annotation_replace_bases`.